### PR TITLE
Update swagger documentation (notification obj) to match 'Outgoing No…

### PR DESCRIPTION
**Pull request by djalters2**
…tification' description on [github MD file](https://github.com/jisc-services/Public-Documentation/blob/master/PublicationsRouter/api/v3/OutgoingNotification.md)

### CODE REVIEW

File: docs/api.yml [hunk 1]
```diff
@@ -500,6 +500,18 @@ components:
     JSONNote:
       type: object
       properties:
+        id: 
+          type: string
+          example: "149f72cb93e345708cf924e432d4f996"
```
Comment (djalters2) [inline docs/api.yml]: Not sure if this is *really* how note id's look

File: docs/api.yml [hunk 2]
```diff
@@ -525,11 +537,20 @@ components:
...
+          type: string
+          example: splash
         format:
           type: string
           example: application/pdf
         url:
           type: string
+        packaging:
+          type: string
+          example: "<package format identifier string>"
```
Comment (djalters2) [inline docs/api.yml]: Real `PF id` better instead?

File: docs/api.yml [hunk 3]
```diff
@@ -500,6 +511,18 @@ components:
...
+        id: 
+          type: string
+          example: "149f72cb93e345708cf924e432d4f996"
+        created_date:
+          type: string
+          format: date
+        analysis_date:
+          type: string
+          format: date-time
+        event:
```
Comment (adam-rehin) [inline docs/api.yml]: As far as I can tell, by examining data in Elasticsearch, `event` is never used (certainly this field doesn't exist in any routed notifications.  
So we should probably say, at least for Outgoing notifications that this is likely to be absent.

File: docs/api.yml [hunk 4]
```diff
@@ -525,11 +548,20 @@ components:
     NoteLink:
       type: object
       properties:
+        type:
+          type: string
+          example: splash
```
Comment (adam-rehin) [inline docs/api.yml]: I think this should probably say <type of content available at the URL, e.g. "fulltext" or "package" or "splash">

File: docs/api.yml (hunk 2 above)
Comment (adam-rehin) [reply docs/api.yml]: Yes, I think so.  

Typical values: "https://pubrouter.jisc.ac.uk/FilesAndJATS" or "http://purl.org/net/sword/package/SimpleZip"

NOte that packaging may be absent, where links.access: "public".
image

File: docs/api.yml [hunk 5]
```diff
@@ -525,11 +548,20 @@ components:
...
           type: string
           example: application/pdf
         url:
           type: string
+        packaging:
+          type: string
+          example: "<package format identifier string>"
+        access:
+          type: string
+          example: router
```
Comment (adam-rehin) [inline docs/api.yml]: Other examples "public", "special".

File: docs/api.yml [hunk 6]
```diff
@@ -693,14 +731,18 @@ components:
         day:
           type: integer
           example: 20
+        season:
+          type: string
+          example: Spring
```
Comment (adam-rehin) [inline docs/api.yml]: Best to quote string examples.

File: docs/api.yml [hunk 7]
```diff
@@ -693,14 +731,18 @@ components:
...
     NoteHistoryDate:
       type: object
       properties:
         date_type:
           type: string
         date:
           type: string
-          format: date
+          example: "2020-03-01"
```
Comment (adam-rehin) [inline docs/api.yml]:
```suggestion
          example: "2020-03-01" or "2020-11" or "2021"
```

File: docs/api.yml [hunk 8]
```diff
@@ -566,9 +598,15 @@ components:
...
-          example: true
+        free2read:
+          type: object
+          properties:
+            start:
+              type: string
+              format: date
+            end:
+              type: string
+              format: date
```
Comment (adam-rehin) [inline docs/api.yml]: Can this have a `description` that makes a note that this value if present should be ignored and may be deprecated in future.

File: docs/api.yml [hunk 9]
```diff
@@ -693,14 +734,19 @@ components:
...
+          description: "Season of publication (for example, Spring, Third Quarter)"
     NoteHistoryDate:
       type: object
       properties:
         date_type:
           type: string
         date:
           type: string
-          format: date
+          example: "'2020-03-01' or '2020-11' or '2021'"
```
Comment (djalters2) [inline docs/api.yml]: Update per suggest change. Looks like this:

image

image

**Pull request merged**